spec.bs

```diff
@@ -118,6 +118,8 @@ partial interface Navigator {
 dictionary AuctionAd {
   required USVString renderURL;
   any metadata;
+  USVString buyerReportingId;
+  USVString buyerAndSellerReportingId;

```

Comment:
[ReportingBrowserSignals](https://github.com/WICG/turtledove/pull/701/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR2850) defines it as a DOMString. It seems this can be DOMString, right?
Same to `buyerReportingId`

Reply:
Nice catch. The impl uses USVString for all.  I don't really have a good sense for broader rules of these, but it feels like making it possible to feed it invalid unicode isn't useful, and since this is in the renderer, not worklet, I don't need to put in the work to get the verification for free...